src/CBT.NuGet/Tasks/GenerateNugetProperties.cs

```diff
@@ -101,7 +100,11 @@ public bool Execute(string packageRestoreFile, string[] inputs, string propsFile
                     Log.LogMessage(MessageImportance.Low, $"NuGet package properties file '{propsFile}' is up-to-date");
                     return true;
                 }
-
+                if (Directory.Exists(packageRestoreFile))
+                {
+                    Log.LogMessage(MessageImportance.Low, $"A directory with the name '{packageRestoreFile}' exist.  Please consider renaming this directory to avoid breaking nuget convention.");
```

Comment:
I left these as just a message because nuget doesn't care only our stuff did.  However it is bad convention so I was spiteful and place a build message.  But in reality I should probably not even have a build message since it isn't my place to force religion.